### ISSUE TITLE
A tag called "data" seems to have special treatment

### ISSUE THREAD

**Issue by GregSutcliffe** [TracksApp/tracks#2659]
Most tags display correctly, to the right of a TODO. But I have a tag "data", and that seems odd, it renders to the left:

![image](https://user-images.githubusercontent.com/827786/132479164-bee72802-d2f6-4b07-9bda-fb98f95dc8b5.png)

This is in the main view, not mobile which renders correctly.

**Comment by ZeiP** [comment]
Yeah, this seems to be because the tag name is also used as a class in the element surrounding the tag:
![screenshot-1631172077](https://user-images.githubusercontent.com/202163/132641116-76eaa2b2-754c-4c19-aecb-5b894dd10227.png)

Not sure if it's actually necessary, maybe we could just remove the class.

**Comment by GregSutcliffe** [comment]
Yeah, opening "Inspect Page" in Firefox and editing it to `class="tag"` doesn't seem to break anything, and fixes the issue. I'll see if I can fix it server-side when I have a moment
